dbt_platform_helper/domain/maintenance_page.py

```diff
@@ -38,20 +38,14 @@ def __init__(
         application_name: str,
         env: str,
         original_exception: Exception,
-        rolled_back_rules: dict[str, bool] = {},
```

Comment:
I assume this defaulted parameter was never used?

Reply:
yea it wasn't, everytime this exception would be raised we would log the rolled back rule types but that is now a part of the standard delete logging